src/hooks/AuditStatus/useAuditLogActions.ts

```diff
@@ -68,11 +73,13 @@ interface AuditLogActionsResponse {
 const useAuditLogActions = ({
   record,
   buttonToogle,
```

Comment:
I missed this in a previous review; please fix the spelling (`Toogle` should be `Toggle`)